src/app/container/info-tab/info-tab.component.html

```diff
@@ -267,14 +267,45 @@
           <!-- Public Topic -->
           <li *ngIf="isPublic">
             <span><strong matTooltip="Short description of the tool">Topic</strong>:&nbsp;</span>
-            <span>{{ tool?.topicSelection === TopicSelectionEnum.AUTOMATIC ? tool?.topicAutomatic : tool?.topicManual }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.AUTOMATIC">{{ tool?.topicAutomatic }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.MANUAL">{{ tool?.topicManual }}</span>
+            <span *ngIf="tool?.topicSelection === TopicSelectionEnum.AI">
+              {{ tool?.topicAI }}
+              <span
+                class="bubble ai-bubble"
+                matTooltip="This topic sentence was AI generated. For more information, please click the bubble to visit our docs"
+              >
+                <img class="ai-icon" src="/assets/svg/ai-icon.svg" alt="AI generation icon" />
+                <span>AI</span>
+                <img class="ai-icon" src="/assets/svg/help.svg" alt="help icon" />
+              </span>
+            </span>
           </li>
           <!-- Private topic Topic -->
           <ng-container *ngIf="!isPublic">
             <li>
               <span><strong matTooltip="Short description of the tool">Topic Automatic</strong>:&nbsp;</span>
               <span>{{ tool?.topicAutomatic }}</span>
             </li>
+            <li>
+              <span
+                ><strong matTooltip="Short description of the tool, from the GitHub repository description generated with AI"
```

Comment:
"from the GitHub repository description" is not applicable. The AI topic is generated from the primary descriptor content, but I don't think the user really needs to know that